kornia/geometry/conversions.py

```diff
@@ -340,7 +340,7 @@ def _compute_rotation_matrix_taylor(angle_axis):
     # create output pose matrix
     rotation_matrix = eye_like(3, angle_axis, shared_memory=False)
     # fill output matrix with masked values
-    rotation_matrix[..., :3, :3] = mask_pos * rotation_matrix_normal + mask_neg * rotation_matrix_taylor
```

Comment:
Then it seems useless line 341?

Reply:
Yes, I dont know why original implementation first made tensor and then modified inplace. Removing line 341 makes sense.